packages/grafana-ui/src/components/Table/DefaultCell.tsx

```diff
@@ -48,14 +47,22 @@ export const DefaultCell = (props: TableCellProps) => {
     }
   }
 
+  const isStringValue = typeof value === 'string';
+
+  const cellStyle = getCellStyle(tableStyles, cellOptions, displayValue, inspectEnabled, isStringValue);
+
+  if (isStringValue && cellProps.style?.justifyContent === 'flex-end') {
```

Comment:
so this a bit wonky because the current API allows passing in arbitrary style, and we see these styles being passed in with the implied assumption that they'll be applied to a container that's `display: flex`, which is kinda leaky. this api can be improved to specify text alignment explicitly (via enum) rather than relying on passed-in raw css props.

i dont think this will be a problem in the short term unless there is some crazy flex-related styling applied to plain text cells, other than for the purpose of text alignment.